### PR TITLE
Fix yarn tests

### PR DESCRIPTION
**Pull request by dallasread**
This PR fixes tests for `search.js.erb`. 

To save an extra network request on the support widget, we inject the articles directly into the production version of `search.js`. As a result, we have to be creative in how we test it.

### CODE REVIEW

**Comment by sbastn** [comment]
@dallasread were the changes on `templates/search.js.erb` done by a linter? The only difference that I spot is the spacing...Did you update anything on this file?

File: templates/search.js.erb [hunk 1]
```diff
@@ -1,121 +1,121 @@
...
+  module.exports = {
+    search: search,
+    applyDictionary: applyDictionary,
+    articleScore: articleScore,
+    loadArticles: loadArticles,
+    fixRelativeImgSrcs: fixRelativeImgSrcs,
+    dictionaryTermMatches: dictionaryTermMatches
+  };
+else {
+  (function () {
```
Comment (dallasread) [inline templates/search.js.erb:114]: I moved this wrapping scope from being around the entire file.

File: content/search/results.js [hunk 2]
```diff
@@ -16,21 +16,28 @@
     var $ul = document.createElement('ul');
 
     $h1.innerText = 'Search results';
+    $ul.className = 'articles-list';
```
Comment (dallasread) [inline content/search/results.js:19]: I noticed that there was already some styling to add nicer bullets to search results.

File: templates/search.js.erb [hunk 3]
```diff
@@ -1,121 +1,127 @@
...
-  var MAX_RESULTS = 30;
-  var MIN_SCORE = 15;
-  var PUNCTUATION = /['";:.()?-]/g;
-  var WHITESPACE = /\s+/;
-  var rootURL = 'https://support.dnsimple.com';
+var ARTICLES = <%= articles %>;
+var DICTIONARY = <%= dictionary %>;
+var MAX_RESULTS = 30;
+var MIN_SCORE = 15;
+var PUNCTUATION = /['";.()?-]/g;
```
Comment (dallasread) [inline templates/search.js.erb:5]: I removed `:` to support our little-know "cat:dns" syntax (the original failing test).

**Comment by dallasread** [comment]
@sbastn There were 2 differences – I should have let you know. I've made comments on them.

**Review by sbastn — APPROVED** [review]

**Pull request merged**